### ISSUE TITLE
Time picker prevents other components from being selected

### ISSUE THREAD

**Issue by aranvir** [h2oai/wave#1896]
#### Wave SDK Version, OS

wave server 0.25.1
h2o-wave 0.25.1
python 3.10.7
windows 10

#### Actual behavior

The last added `ui.time_picker` hogs the focus for the form. As long as it was not clicked once, other components cannot be used. One does not have to select a value, just clicking and dismissing the time picker releases the focus and other components become responsive again.

Elements like the dropdown menu flicker briefly, when clicked. So focus seems to go to the dropdown menu for a short time and is then put back on the time picker.

#### Expected behavior

No component should keep focus from others.

#### Steps To Reproduce

1. Run the app below
2. Try and click on the dropdown menu. It will not show up.
3. Try and click on the first time picker. It will not show up.
4. Try and click on the second time picker. It will show up. Select a value or dismiss it.
5. Now the other 2 components are responsive again.

Refresh the page to restore the flawed state.

```python
from h2o_wave import main, app, Q, ui


@app('/demo')
async def serve(q: Q):
    items = [
        ui.dropdown(name='dropdown', label='Dropdown', value="", width='300px', choices=[
            ui.choice(name='choice1', label='choice1'),
            ui.choice(name='choice2', label='choice2'),
            ui.choice(name='choice3', label='choice3'),
        ]),
        ui.time_picker(name='timepicker', value="", label="Standard time picker"),
        ui.time_picker(name='timepicker2', value="", label="Standard time picker 2"),
    ]
    q.page['example'] = ui.form_card(box='1 1 4 8', items=items)
    await q.page.save()
```

**Comment by mturoci** [comment]
@marek-mihok please have a look at this as a high prio.

**Comment by mturoci** [comment]
Released 0.25.2 where this should be resolved. @aranvir can you please check it out and let me know if works for you? Thanks!

**Comment by aranvir** [comment]
Works! Thanks for the quick fix, you guys are amazing!
